src/main/utils/upload.ts

```diff
@@ -102,33 +97,24 @@ export async function getElectronFile(filePath: string): Promise<ElectronFile> {
 export const setToUploadFiles = (
     filePaths: string[],
     collectionName: string,
-    collectionIDs: number[],
     done: boolean
 ) => {
     store.set('done', done);
```

Comment:
why do need this `done` property, we can just check if `filePaths.length>0 ` to know if we have pending uploads